### DISCUSSION TITLE
How to uninstall?

### DISCUSSION THREAD

**Discussion by FindtheDoor** [Alex313031/thorium#726] [General]
How do I delete this from my system? I deleted the folders themselves, but Thorium still shows under default apps in Windows? I want it completely deleted from my system.

**Comment by gz83** [comment]
You can uninstall it through the control panel

**Reply by FindtheDoor** [reply]
It was the portable. I deleted the files but when I did it still shows under default apps in Windows 11

**Comment by gz83** [comment]
Close discussions that are outdated and inactive.
